src/Templates/NIPSWeb/live.twig

```diff
@@ -23,7 +23,7 @@
             <p>Connecting to server...</p>
             <div id="init-progressbar"></div>
         </div>
-        <div class="ui-dialog-buttonpane ui-widget-content ui-helper-hidden" id="init-overlay-fallback">
+        <div class="ui-dialog-buttonpane ui-widget-content hidden" id="init-overlay-fallback">
```

Comment:
this file hasn't been updated and doesn't match main.twig for show planner - tho I'm guessing it doesn't work atm anyway